DabraWarq/DWdavies01.xml

```diff
@@ -777,20 +780,19 @@ also catalogued by Jeremy Brown for the PEMM project, see, <ref
                     <title type="complete" ref="LIT4274AkkonuB#Stanza32"/>
                     <textLang mainLang="gez"/>
                   </msItem>
+                </msItem>
                   <msItem xml:id="ms_i1.4">
```

Comment:
the nesting is still wrong as 1.4 is nested inside of 1.3, but everything else looks good! just take 1.4 out

Reply:
do not forget to click on the Oxygen alignment button to see the nesting better

<img width="201" height="53" alt="image" src="https://github.com/user-attachments/assets/a6a909f1-bb28-40d0-8c63-69fefc60fe9a" />